scapy/contrib/automotive/someip.py

```diff
@@ -1,3 +1,8 @@
+# SPDX-License-Identifier: GPL-2.0-only
+# This file is part of Scapy
+# See https://scapy.net/ for more information
+# Copyright (C) Sebastian Baar <sebastian.baar@gmx.de>
+
 # MIT License
```

Comment:
What is the correct license?

Reply:
How about using
```
# SPDX-License-Identifier: GPL-2.0-only OR MIT
```
(it's part of [the format](https://spdx.dev/ids/#Examples:~:text=How%20to%20use-,Examples,-The%20examples%20below))

Reply:
We added the MIT license, since we started our SOME/IP development based on a non-mainline (Scapy) SOME/IP project which was released with MIT license. Therefore, this "second" license ended up in the file.